File: Techschool/Techschool.BLL/Models/Vacations/AnnualVacationModel.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Techschool.BLL.Models.Vacations
{
    public class AnnualVacationModel : VacationModel
    {
        public string OrderNumber { get; set; }
        public DateTime OrderDate { get; set; }
    }
}

File: Techschool/Techschool/ClientApp/src/app/models/vacations/annual-vacation.model.ts
import { VacationModel } from './vacation.model';

export class AnnualVacationModel extends VacationModel {
  startOfVacationDate: Date;
  endOfVacationDate: Date;
  orderNumber: string;
  orderDate: Date;
  annualVacationFormId: string;
}
